core/report-l10n.rb

```diff
@@ -170,7 +170,7 @@ module ISO639
 dar|||Dargwa|dargwa
 day|||Land Dayak languages|dayak, langues
 del|||Delaware|delaware
-den|||Slave (Athapascan)|esclave (athapascan)
+den|||Slave (Athapascan)|agent (athapascan)
```

Comment:
Nope.

https://en.wikipedia.org/wiki/Slavey_language

> Slavey (also Slave, Slavé) is an Athabaskan language

Reply:
@daniel-beck Removed the change, then did an interactive rebase. Could you resolve? Thanks.